renovate.json

```
@@ -48,6 +48,12 @@
         "ModSecurity2"
       ],
       "allowedVersions": "/^v2.*/"
+    },
+    {
+      "matchDepNames": [
+        "nginxinc/nginx-unprivileged"
+      ],
+      "allowedVersions": "/^[0-9]+\\.[0-9]*[02468]\\.[0-9]+$/"
```

Your expression will also match 1.0000003.4. Granted, it's not super important as we can expect the version numbers to be correct :). Still, here's an improved version:
```suggestion
      "allowedVersions": "/^[0-9]+\\.(?:[1-9]\\d*)?[02468]\\.[0-9]+$/"
```

---

This regex is more strict. I fixed it. Thanks for the review!